atlasaction/testdata/schema.hcl

```diff
@@ -1,8 +1,8 @@
 table "t1" {
-  schema = schema.public
+  schema = schema.main
   column "c1" {
     type = int
   }
 }
 
-schema "public" {}
\ No newline at end of file
+schema "main" {}
```

Comment:
not related to this pr, but integration of `schema/test` was failing:

![image](https://github.com/user-attachments/assets/c759520c-e307-414d-aeff-f544ebc425d0)